### PR TITLE
Adding Hardware Accelerating Sub-Section

### PR DESCRIPTION
**Pull request by DevilsDesigns**
added a sub section Hardware Acceleration
- Adding a Sub file inside called ubuntu-debian-hardware-acceleration.md
- this is for Debian and Ubuntu OS Distros
- im using Kubuntu 20.04
- This is a full walkthrough on how to do install/setup Hardware Accelerating with NVENC from start to finsh

### CODE REVIEW

File: general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                |                               |                      |               MIG M. |
+                |===============================+======================+======================|
+                |   0  GeForce GTX 165...  Off  | 00000000:01:00.0  On |                  N/A |
+                |  0%   50C    P0    26W / 100W |    719MiB /  3895MiB |     17%      Default |
+                |                               |                      |                  N/A |
+                +-------------------------------+----------------------+----------------------+
+
+   2. So to find the correct drivers select from the dropdown. Againb using my Geoforce 970 as an example
+
+        Product Type: Geoforce (whoever manufacters your GPU)
```
Comment (cromefire) [inline general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md:69]: I guess you you meant geforce

File: general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        Product Series: Geoforce 900 Series (make sure you are not selecting the M unless your on a laptop)
+        Product Geoforce GTX 970 (Select what model of graphics card you have for your GPU)
+        Operating System: LINUX 64BIT        
+        Download Type: 
+        
+        **(Important)**  Select Linux Short Lived if your graphics card is a bit older.WHEN I SAY OLDER I MEAN 2 YEARS OR LESS.
```
Comment (cromefire) [inline general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md:75]: Should this be a fixed year (like 2018 in this case) or always 2 years, relative to when the user reads it?

File: general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        **(Important)**  Select Linux Long Lived if your NVIDIA CARD IS FAIRLY NEW.
```
Comment (cromefire) [inline general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md:76]: Also less caps please

File: general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+**This is extremely important. If you want to have more than 2 concurrent streams running on your jellyfin server i would HIGHLY Recommend patching your existing drivers below. If you are not going to have more than 2 consecutive streams running at once please skip to the Configuring Your Settings in Jellyfin.**
+    
+   1. Now go to this link for patching the drivers.   [NVIDIA Github patch](https://github.com/keylase/nvidia-patch) 
+
+   2. Scroll Down to the versions table or type in browser CTRL + F. Then type in your browser window for your drive
+        - IE. Geoforce GTX 970 = 432.21 (it showed us the driver on nvidias website)
+     
+   3. Once youve located your correct driver hover over the driver link with your mouse right click the link and copy the link location.
+
+   4. **(Important) YOU NEED TO RUN THESE COMMANDS IN CLI NOT IN YOUR DESKTOP TERMINAL**
```
Comment (cromefire) [inline general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md:90]: Again same thing as in the other PR

File: general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+        - IE. my user is `devilscoder`      
+        - so i login with user `devilscoder` and the password associated with it.
+
+   6. Now make the this directory and navigate to it by sending this command.
+   
+                sudo mkdir /opt/nvidia && cd /opt/nvidia
+
+   7. Now you need to download the correctly update driver from the command posted below
+       
+                sudo wget https://international.download.nvidia.com/XFree86/Linux-x86_64/(driver firmware number)/NVIDIA-Linux-x86_(driver firmware number).run
```
Comment (cromefire) [inline general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md:104]: No `sudo wget` please downloading stuff with root is highly discouraged

File: general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+        
+                `bash ./patch.sh`
+    
+   12. Now it should display succesfully patched. Now that everythings done you can restart xServer or whatever you were using for DE (Desktop Enviroment)    
+        - To start your xserver run    
+                `startx`
+        - Your Desktop Enviroment should then display login and move your way to your jellyfin install in browser
+            
+###  Configuring Your Settings in Jellyfin
+   1. Open up your jellyfin Web UI by typing in one of the following domains.
```
Comment (cromefire) [inline general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md:142]: I guess if the user has got so far to use this guide you can assume he can open the web UI

File: general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+        - ie. (if you have a reverse-proxy running) https://sub.yourdomain.com        https://stream.jellyfin.com
+
+   2. Once there click on the top left hamburger menu and click on dashboard in the menu
+        So because these are enabled please write them down somewhere.
+   3. Once Loaded click on the hamburger menu again and select playback
+   
+   4. Now Open this link [NVENC/DECODING SETTINGS FOR GPU](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix#Encoder)   
+        - Once there if your card is 2 years or older it will most likely not have NVENC (NVIDIA ENCODING) on the card. So lets go  scroll to the bottom and select the green button that says Geoforce/Titan becuase mine in this example is a Geoforce GTX 970.
+        - Now it should pull up more graphics card. Hit the key combinations CTRL + F and type your graphics card so im going to type 970. The page will take you to your card and now check the settings mine has these decoding features.
+                `MPEG-1   YES      MPEG-2   YES      VC-1   YES      H.264(AVCHD)   YES`
```
Comment (cromefire) [inline general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md:154]: Add the note about required h264 encoding should be somewhere here

File: general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+        - So because these are enabled please write them down somewhere.
+   
+   5. Now go back to Jellyfin and click the first dropdown menu and select Nvidia NVENC
+
+   6. Then enable only those options. That you wrote down in step 19. So Since my card does not encoding i need to disable anything with encoding in the options if you dont bascically you wont be able to watch your videos on your Jellyfin server.
```
Comment (cromefire) [inline general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md:159]: There's no step 19, also typos

**Comment by cromefire** [comment]
You should also probably link this from the other guide

**Comment by MrTimscampi** [comment]
To the risk of sounding a bit mean, I have quite a few issues with this:

* It's badly written: It reads like a forum post and not like documentation. Uses of the first person, bad punctuation, inconsistent use of capital letters, ...
* Bad practices: You shouldn't install your driver through Nvidia's website. Go through your distribution's repositories. Ubuntu has the drivers in their repos. You can hold packages to prevent them from upgrading automatically.
* Confusing formatting: It doesn't make use of any of DocFX's formatting features (Code blocks, warning/info/danger blocks, etc) making for a hard read and quite ugly presentation.
* It's basically "Run these commands" without any explanation of why, what they do or the risks involved in running arbitrary commands from a website.
* Imo, it runs into the same issue that resulted in us not wanting any tutorial for reverse proxies and keeping the documentation there specific to Jellyfin: there's a lot of grounds to cover for various kinds of configurations and it's not our job to do so. Users are expected to research by themselves and know how to administer their systems before attempting to use Jellyfin. It's not our job to teach them how to install a driver on their OS.

As usual, I'll defer to @jellyfin/core for a decision, but personally I don't think we want this kind of "hand-holding" tutorials in our documentation, even with the style issues fixed.

**Comment by joshuaboniface** [comment]
I agree wiht @MrTimscampi's assessment. I'm not a fan of the PR in its current state. @cromefire's feedback is a good start but this does not read like formal documentation. It could get there with a lot of polish but I'd suggest reading some of the rest of the docs and rewriting it to fit the style, tone, and formatting they display.

**Comment by joshuaboniface** [comment]
Closing based on my last comment.

**Pull request closed without merging**